File: 3830-find-closest-person/3830-find-closest-person.py
class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:
        timeone=abs(x-z)
        timesecond=abs(y-z)
        if(timeone<timesecond):
            return 1
        elif(timeone>timesecond):
            return 2
            
        return 0

File: 3830-find-closest-person/3830-find-closest-person.cpp
class Solution {
public:
    int findClosest(int x, int y, int z) {
        int time1=abs(x-z);
        int time2=abs(z-y);
        if(time1<time2){
            return 1;
        }
        else if(time1>time2){
            return 2;
        }
        return 0;
        
    }
};